### PR TITLE
SAK-31999: allow one to copy a list of site member's usernames in Site Info

### PR DESCRIPTION
**Pull request by nicholaswilson100**

### CODE REVIEW

File: site-manage/site-manage-util/util/src/java/org/sakaiproject/site/util/SiteConstants.java [hunk 1]
```diff
@@ -33,6 +33,8 @@
 	
 	public static final String SORTED_BY_GROUP_TITLE = "group_title";
 	
+	public static final String SORTED_BY_DISPLAY_ID = "display_id";
```
Comment (ern) [inline site-manage/site-manage-util/util/src/java/org/sakaiproject/site/util/SiteConstants.java]: This constant should be more aptly named like "SORTED_BY_PARTICIPANT_DISPLAY_ID" as its sorting participants.

File: site-manage/site-manage-tool/tool/src/webapp/vm/sitesetup/chef_site-exportMemberList.vm [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+ <h3>Participant List for: $validator.escapeHtml($!siteTitle)</h3>
+ <div>Total: $participantListSize</div>
+#if ($!allowUpdate || $allowUpdateSiteMembership || $!viewRoster)
+  <form name="participantForm" id="participantForm" action="#toolForm("SiteAction")" method="post">
+   <table class ="listHier lines nolines"  cellpadding="0" cellspacing="0" border="0" summary ="$tlang.getString("sitegen.siteinfolist.partlist.summary")">
+    <tr>
+     <th id="id">
+      <a href="#toolLinkParam("SiteAction" "doSort_roster" "criterion=display_id")" title ="$tlang.getString("sitegen.siteinfolist.sortid")"> 
+       $tlang.getString("sitegen.siteinfolist.id")
+       #if ($currentSortedBy.equals("display_id"))
```
Comment (ern) [inline site-manage/site-manage-tool/tool/src/webapp/vm/sitesetup/chef_site-exportMemberList.vm]: Following the existing pattern should be "participant_display_id".

File: site-manage/site-manage-util/util/src/java/org/sakaiproject/site/util/SiteConstants.java (hunk 1 above)
Comment (adrianfish) [inline site-manage/site-manage-util/util/src/java/org/sakaiproject/site/util/SiteConstants.java]: Agree with Earle. GROUP_TITLE then PARTICIPANT_DISPLAY_ID.

File: site-manage/site-manage-tool/tool/src/webapp/vm/sitesetup/chef_site-exportMemberList.vm (hunk 2 above)
Comment (adrianfish) [inline site-manage/site-manage-tool/tool/src/webapp/vm/sitesetup/chef_site-exportMemberList.vm]: Agreee with Earle. If display_id is the participant, then make it clear by using 'participant_display_id'.

**Comment by nicholaswilson100** [comment]
I think I've addressed all the comments in this PR that need addressing.

**Comment by adrianfish** [comment]
Not working for me. I picked onto a fresh master, rebuilt site-manage and when I click the button I get this:

screen shot 2017-03-14 at 19 42 46

**Comment by nicholaswilson100** [comment]
@adrianfish : Actually I don't really like the way this person has done this too much at all.  There's no need to have the pop up include the portal, it's unnecessary code and more to maintain.  It's much easier if it was just a link you clicked that gave you a text file. Easy to do and worth a rewrite imho.

**Comment by nicholaswilson100** [comment]
I'd rather we spent our time implementing a nicer solution than fixing up a bad and buggy one.

**Comment by adrianfish** [comment]
Close the PR and just keep the branch. You won't have time to redo this before 12.x, unless you like working late :)

**Pull request closed without merging**